action/payment/routes.go

```diff
@@ -13,8 +13,8 @@ type payment struct {
 	RazorpaySignature string `gorm:"column:razorpay_signature" json:"razorpay_signature" validate:"required"`
 }
 
-// Router - Group of payment router
-func Router() chi.Router {
+// UserRouter - Group of payment router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /payments - return list of payments
```

Comment:
UserRouter must access *GET /payments/{payment_id}* also

Reply:
It is already there at line 24